internal/platform/web/web.go

```diff
@@ -107,3 +108,28 @@ func (a *App) Handle(verb, path string, handler Handler, mw ...Middleware) {
 	// Add this handler for the specified verb and route.
 	a.TreeMux.Handle(verb, path, h)
 }
+
+// Shutdown is a type used to help with the graceful termination of the service.
```

Comment:
Should this start lowercase `shutdown is ...`.

Reply:
Brilliant catch